src/cancellable-fetch.ts

```diff
@@ -31,10 +31,15 @@ export const cancellableFetch = <T>(
 ): CancellableFetch<T> => {
   const abortController = new AbortController();
   const abort = () => abortController.abort();
+  const bearerToken = process.env.REACT_BEARER_TOKEN;
 
   const fetchPromise = fetch(url, {
     ...init,
-    headers: { ...init?.headers, Accept: 'application/json' },
+    headers: {
+      ...init?.headers,
+      Accept: 'application/json',
+      ...(bearerToken ? { Authorization: `Bearer ${bearerToken}` } : {}),
```

Comment:
Since the request will go through `/proxy/`, the `Authorization` header will be added automatically by the proxy. In other words, we shouldn't need to pass the bearer token from the frontend (see https://github.com/openshift/enhancements/blob/master/enhancements/console/dynamic-plugins.md#delivering-plugins).

Reply:
This is for local development.

Reply:
👍